scripts/test_updates.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+PGTEST_TMPDIR=${PGTEST_TMPDIR:-/tmp}
+UPDATE_PG_PORT=${UPDATE_PG_PORT:-6432}
+CLEAN_PG_PORT=${CLEAN_PG_PORT:-6433}
+
+UPDATE_FROM_TAG=${UPDATE_FROM_TAG:-0.1.0}
+
+wait_for_pg () {
+set +e
+for i in {1..10}; do
+  sleep 2
+
+  pg_isready -h localhost -p $1
+
+  if [[ $? == 0 ]] ; then
+    set -e
+    return 0
+  fi
+done
+exit 1
+}
+
+docker rm -f timescaledb-orig timescaledb-updated timescaledb-clean || true
```

Comment:
Is docker the only way we can run this test? I think we should try to keep docker optional also for development (like it currently is).